### PR TITLE
Blocks height link for defi scan

### LINKED ISSUE #948: `network details` - block height details
#### What would you like to be added:
In network details, we show the block height. We should make it clickable and redirect to DeFiScan block details page.

https://defiscan.live/blocks/1228320

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Make block height clickable and redirect to DeFiScan block details page.

#### Which issue(s) does this PR fixes?:

Fixes #948

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

**Comment by fullstackninja864** [comment]
SS of implementation 
image

**Review by thedoublejay — CHANGES_REQUESTED** [review]
Make the block number pink, check Transaction URL in Ocean Interface as reference.

**Comment by kyleleow** [comment]
Since it is clickable, might want to add an e2e test to it?

**Comment by fullstackninja864** [comment]
@kyleleow  , Thanks for suggestion, I have added e2e for the block height link.

File: mobile-app/app/contexts/DeFiScanContext.tsx [hunk 1]
```diff
@@ -58,3 +65,8 @@ function getTxURLByNetwork (network: EnvironmentNetwork, txid: string, rawtx?: s
 
   return baseUrl
 }
+
+function getBlocksURLByNetwork (network: EnvironmentNetwork, blockCount: number): string {
+  const baseUrl = `https://defiscan.live/blocks/${blockCount}${getNetworkParams(network)}`
```
Comment (thedoublejay) [inline mobile-app/app/contexts/DeFiScanContext.tsx:70]: Can we store the URL as a constant on this file?

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/contexts/DeFiScanContext.tsx (hunk 1 above)
Comment (thedoublejay) [reply mobile-app/app/contexts/DeFiScanContext.tsx:70]: URL is used on getBlocksURL and getTxURL

Comment (fullstackninja864) [reply mobile-app/app/contexts/DeFiScanContext.tsx:70]: I will do that in next PR for tokens where we will use a common method to get the url

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**